spec/features/associating_topics_to_mainstream_browse_pages_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "spec_helper"
+
+# NOCOMMIT: Don't commit this until it's passing!
+describe "associating topics to mainstream browse pages" do
+  before do
+    stub_user.permissions << "GDS Editor"
+    stub_all_panopticon_tag_calls
+  end
+
+  context "existing mainstream browse pages" do
+    let(:mainstream_browse_page) { create(:mainstream_browse_page) }
+    let(:topic)                  { create(:topic) }
+    let(:topic_two)              { create(:topic) }
+
+    before do
+      mainstream_browse_page.save
+      topic.save
+      topic_two.save
```

Comment:
You don't need this block if you replace the above `let` calls with `let!` calls.  `let` blocks are lazily evaluated, so the models won't be created until the method is first called.  As the code stands, it's actually doing a double-save (once from the create call when the variable is referenced, and once from the explicit save call.)